licenses.d2iq.yaml

```diff
@@ -618,3 +615,18 @@ resources:
       - url: https://github.com/mirror/busybox
         ref: master
         license_path: LICENSE
+  - container_image: gcr.io/kubecost1/cost-model:prod-1.106.7
+    sources:
+      - url: https://github.com/opencost/opencost
+        ref: v${image_tag#prod-}
+        license_path: LICENSE
+  - container_image: ghcr.io/mesosphere/dkp-container-images/gcr.io/kubecost1/frontend:prod-1.106.7-d2iq.0
+    sources:
+      - url: https://github.com/opencost/opencost
+        ref: v1.106.7
+        license_path: LICENSE
+  - container_image: gcr.io/google_containers/pause:3.2
```

Comment:
Why do we have such an old pause image here?

Reply:
```
 + ./services/nkp-insights/1.2.0/helmrelease/helmrelease.yaml

docker.io/aquasec/kube-bench:v0.7.2
docker.io/bitnami/kubectl:1.29.6
docker.io/mesosphere/insights:v1.2.0-beta.3
docker.io/mesosphere/trivy-bundles:0.53.0-20240726T101319Z
gcr.io/google_containers/pause:3.2
```

I'll create insights ticket to bump it.

Reply:
will get bumped in the `dkp-insights` to `registry.k8s.io/pause:3.10`